db/migrate/20231026163527_create_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUser < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
[REQUIRED] I think it is better to use text instead of string since. text can store an unlimited number of characters and bio tends to be longer than texts like name or URL for example.

Reply:
Thank you for your review @fobadara , I have made the required changes.

Reply:
Fixed.